app/models/concerns/tristateable.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Reads a column, bypassing activerecord field accessors, so that we can
+# differentiate between nil values and boolean false (by default nils in
+# activerecord are converted to false). Naturally, booleans can only hold
+# true/false, so these fields should be created as strings.
+#
+module Tristateable
```

Comment:
Are we sure that we still need the Tristateable? The DS will populate all booleans with True/False/Nil - so you only need to differentiate between nil and a boolean T/F.